app/views/projects/_project.html.haml

```diff
@@ -15,6 +15,8 @@
         - if project.budget
           .project-budget
             = t(:budget_html, budget_left: project.budget_status, budget: project.budget)
+        - if project.description
+          %p= markdown(project.description)
```

Comment:
Seems good, but the markdown formatting could be improved.

Maybe this should be a new issue?

![final rails project hours 2015-10-20 09-59-39](https://cloud.githubusercontent.com/assets/2607/10609361/51f37b02-7711-11e5-8625-636f9d0f5eaa.png)